helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -426,9 +426,9 @@ protected ClusterControllerManager startController(String cluster) {
       }
       workflows.put(WORKFLOW_PREFIX + i, workflow.build());
       WorkflowContext workflowContext = TaskTestUtil
-          .buildWorkflowContext(WORKFLOW_PREFIX + i, TaskState.IN_PROGRESS,
+          .buildWorkflowContext(WORKFLOW_PREFIX + i, TaskState.FAILED,
```

Comment:
Since the test we are checking the accessor, this has been changed to FAILED. In this case, the status of the workflow will be unchanged and the controller does not change it. Hence, the job accessor test would work correctly.